tests/rbd/rbd_utils.py

```diff
@@ -570,25 +570,25 @@ def initial_rbd_config(**kw):
         if not kw.get("config"):
             kw["config"] = {
                 "rep_pool_config": {
-                    "pool": rbd_reppool.random_string(),
-                    "image": rbd_reppool.random_string(),
+                    "pool": "rbd_test_pool_replicated",
```

Comment:
@VasishtaShastry Can we please add a prefix_random so that we don't end up using same pool and image names when we run tests in parallel.

Reply:
Done.